pom.xml

```diff
@@ -6,7 +6,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.taboola</groupId>
     <artifactId>backstage-api-java-client</artifactId>
-    <version>1.1.8.5</version>
+    <version>1.2.0.0</version>
```

Comment:
please use 1.1.8.6.. current release version is 1.1.8.. I just increase the last digit for our internal version.

Reply:
Done